### DISCUSSION TITLE
Issues in Importing map from Roadrunner on Carla 0.9.11

### DISCUSSION THREAD

**Discussion by bg-sumu** [carla-simulator/carla#7057] [Q&A]
OS: Ubuntu 20.04 , Carla version: 0.9.11(Linux build, no Docker)
I created a map on Roadrunner and exported it into .xodr and .fbx files and placed it in Import directory of Carla. Then ran the command make Import. 
But the traffic light poles and the street light poles are not shown properly. What should i do now ?

**Comment by Jinviz** [comment]
Once you have properly included traffic light data in the .xodr file, it would be a good idea to make sure that the traffic light blueprint is properly assigned to the traffic light groups within the editor. I hope this answer will be helpful
